### ISSUE TITLE
Mac tar ball seem to be damaged

### ISSUE THREAD

**Issue by Sebadier** [neopostmodern/wohnungsbot#56]
I am not able to unpack Wohnungsbot-1.3.4-mac.tar.gz. 
I tried with clicking in Finder and "tar -xvfz" and get errors every time. So is the archive damaged or am I doing something wrong?

**Comment by neopostmodern** [comment]
(ノಠ益ಠ)ノ彡┻━┻

I thought I had finally fixed it. What's your OS version?

**Comment by Sebadier** [comment]
Thanks for getting back to me so quickly!
My OS is Catalina (10.15.7 with all the latest updates). 
I can also try to unpack on my linux machine

**Comment by neopostmodern** [comment]
I've confirmed on a borrowed macOS system that the patch I included is indeed not applied for the automatically uploaded archive, but _is_ applied in the version uploaded to the [action run](https://github.com/neopostmodern/wohnungsbot/actions/runs/571220450).
TL;DR I've replaced the archive in the release and it should work now (although it's unfortunately not automatic), please re-download and try again.

**Comment by Sebadier** [comment]
It's working now. Thank you so much for your good work! :)
